packages/metamask-snap/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@sei-js/metamask-snap",
-	"version": "1.0.1",
+	"version": "1.0.2",
```

Comment:
this should automatically get updated by `changesets` so don't think u need to manually bump it